style.css

```diff
@@ -1 +1,69 @@
-/*  Add your styling here */
+
+/* Universal styles */
+
+* {
+    font-family: 'Pacifico', cursive;
+    background-color: rgb(188, 243, 226);
+}
+
+ul {
+    display: flex;
+    justify-content: center;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: rgb(200, 144, 233);
+}
+
+h1, h3 {
+    color: rgb(255,128,153);
+    text-align: center;
+}
+
+/* Navigation styles */
+
+#header-image {
+    height: 150px;
+    width: 150px;
+    display: block;
+    margin: 0 auto;
+}
+
+.navigation li {
+    padding-right: 50px;
+}
+
+/* About section styles */
+
+.about {
+    padding: 100px;
+}
+
+/* Example cake styles */
+
+.example-cakes ul {
+   justify-content: space-around;
```

Comment:
Watch out for default styling on `ul` - inspect this element and you'll see some padding on the left which is setting everything off center. This is a default rule that the browser adds - you can overwrite it with `padding-left: 0`